### ISSUE TITLE
Unsupported layers 

### ISSUE THREAD

**Issue by koflp** [NVIDIA-AI-IOT/torch2trt#644]
Hi dear torch2trt team

I am trying to convert the RAFT model (https://github.com/princeton-vl/RAFT) into trt to check the speed. I encounter following log, which is clear that something is not supported. Any hints to overcome that? 

Warning: Encountered known unsupported method torch.is_autocast_enabled
Warning: Encountered known unsupported method torch.set_autocast_enabled
Warning: Encountered known unsupported method torch.autocast_increment_nesting

Following that I get following errors:

[10/07/2021-16:49:44] [TRT] [E] 3: [layers.h::nvinfer1::ConcatenationLayer::setAxis::624] Error Code 3: Internal Error (Parameter check failed at: layers.h::nvinfer1::ConcatenationLayer::setAxis::624, condition: axis >= 0 && axis < Dims::MAX_DIMS
)
[10/07/2021-16:49:44] [TRT] [E] 3: [CONVOLUTION #1] torch.nn.Conv2d.forward(Conv2d(3, 32, kernel_size=(7, 7), stride=(2, 2), padding=(3, 3)), tensor(shape=[2, 3, 184, 320], dtype=torch.float32)):kernel weights has count 4704 but 9408 was expected
[10/07/2021-16:49:44] [TRT] [E] 4: [CONVOLUTION #1] torch.nn.Conv2d.forward(Conv2d(3, 32, kernel_size=(7, 7), stride=(2, 2), padding=(3, 3)), tensor(shape=[2, 3, 184, 320], dtype=torch.float32)): count of 4704 weights in kernel, but kernel dimensions (7,7) with 6 input channels, 32 output channels and 1 groups were specified. Expected Weights count is 6 * 7*7 * 32 / 1 = 9408
[10/07/2021-16:49:44] [TRT] [E] 4: [convolutionNode.cpp::nvinfer1::builder::ConvolutionNode::computeOutputExtents::39] Error Code 4: Internal Error ([CONVOLUTION #1] torch.nn.Conv2d.forward(Conv2d(3, 32, kernel_size=(7, 7), stride=(2, 2), padding=(3, 3)), tensor(shape=[2, 3, 184, 320], dtype=torch.float32)): number of kernel weights does not match tensor dimensions)
[10/07/2021-16:49:44] [TRT] [E] 3: [CONVOLUTION #1] torch.nn.Conv2d.forward(Conv2d(3, 32, kernel_size=(7, 7), stride=(2, 2), padding=(3, 3)), tensor(shape=[2, 3, 184, 320], dtype=torch.float32)):kernel weights has count 4704 but 9408 was expected
[10/07/2021-16:49:44] [TRT] [E] 4: [CONVOLUTION #1] torch.nn.Conv2d.forward(Conv2d(3, 32, kernel_size=(7, 7), stride=(2, 2), padding=(3, 3)), tensor(shape=[2, 3, 184, 320], dtype=torch.float32)): count of 4704 weights in kernel, but kernel dimensions (7,7) with 6 input channels, 32 output channels and 1 groups were specified. Expected Weights count is 6 * 7*7 * 32 / 1 = 9408
[10/07/2021-16:49:44] [TRT] [E] 4: [convolutionNode.cpp::nvinfer1::builder::ConvolutionNode::computeOutputExtents::39] Error Code 4: Internal Error ([CONVOLUTION #1] torch.nn.Conv2d.forward(Conv2d(3, 32, kernel_size=(7, 7), stride=(2, 2), padding=(3, 3)), tensor(shape=[2, 3, 184, 320], dtype=torch.float32)): number of kernel weights does not match tensor dimensions)
[10/07/2021-16:49:44] [TRT] [E] 3: [CONVOLUTION #1] torch.nn.Conv2d.forward(Conv2d(3, 32, kernel_size=(7, 7), stride=(2, 2), padding=(3, 3)), tensor(shape=[2, 3, 184, 320], dtype=torch.float32)):kernel weights has count 4704 but 9408 was expected
[10/07/2021-16:49:44] [TRT] [E] 4: [CONVOLUTION #1] torch.nn.Conv2d.forward(Conv2d(3, 32, kernel_size=(7, 7), stride=(2, 2), padding=(3, 3)), tensor(shape=[2, 3, 184, 320], dtype=torch.float32)): count of 4704 weights in kernel, but kernel dimensions (7,7) with 6 input channels, 32 output channels and 1 groups were specified. Expected Weights count is 6 * 7*7 * 32 / 1 = 9408
[10/07/2021-16:49:44] [TRT] [E] 4: [convolutionNode.cpp::nvinfer1::builder::ConvolutionNode::computeOutputExtents::39] Error Code 4: Internal Error ([CONVOLUTION #1] torch.nn.Conv2d.forward(Conv2d(3, 32, kernel_size=(7, 7), stride=(2, 2), padding=(3, 3)), tensor(shape=[2, 3, 184, 320], dtype=torch.float32)): number of kernel weights does not match tensor dimensions)
[10/07/2021-16:49:44] [TRT] [E] 3: [CONVOLUTION #1] torch.nn.Conv2d.forward(Conv2d(3, 32, kernel_size=(7, 7), stride=(2, 2), padding=(3, 3)), tensor(shape=[2, 3, 184, 320], dtype=torch.float32)):kernel weights has count 4704 but 9408 was expected
[10/07/2021-16:49:44] [TRT] [E] 4: [CONVOLUTION #1] torch.nn.Conv2d.forward(Conv2d(3, 32, kernel_size=(7, 7), stride=(2, 2), padding=(3, 3)), tensor(shape=[2, 3, 184, 320], dtype=torch.float32)): count of 4704 weights in kernel, but kernel dimensions (7,7) with 6 input channels, 32 output channels and 1 groups were specified. Expected Weights count is 6 * 7*7 * 32 / 1 = 9408
[10/07/2021-16:49:44] [TRT] [E] 4: [convolutionNode.cpp::nvinfer1::builder::ConvolutionNode::computeOutputExtents::39] Error Code 4: Internal Error ([CONVOLUTION #1] torch.nn.Conv2d.forward(Conv2d(3, 32, kernel_size=(7, 7), stride=(2, 2), padding=(3, 3)), tensor(shape=[2, 3, 184, 320], dtype=torch.float32)): number of kernel weights does not match tensor dimensions)
[10/07/2021-16:49:44] [TRT] [E] 3: [CONVOLUTION #1] torch.nn.Conv2d.forward(Conv2d(3, 32, kernel_size=(7, 7), stride=(2, 2), padding=(3, 3)), tensor(shape=[2, 3, 184, 320], dtype=torch.float32)):kernel weights has count 4704 but 9408 was expected
[10/07/2021-16:49:44] [TRT] [E] 4: [CONVOLUTION #1] torch.nn.Conv2d.forward(Conv2d(3, 32, kernel_size=(7, 7), stride=(2, 2), padding=(3, 3)), tensor(shape=[2, 3, 184, 320], dtype=torch.float32)): count of 4704 weights in kernel, but kernel dimensions (7,7) with 6 input channels, 32 output channels and 1 groups were specified. Expected Weights count is 6 * 7*7 * 32 / 1 = 9408
[10/07/2021-16:49:44] [TRT] [E] 4: [convolutionNode.cpp::nvinfer1::builder::ConvolutionNode::computeOutputExtents::39] Error Code 4: Internal Error ([CONVOLUTION #1] torch.nn.Conv2d.forward(Conv2d(3, 32, kernel_size=(7, 7), stride=(2, 2), padding=(3, 3)), tensor(shape=[2, 3, 184, 320], dtype=torch.float32)): number of kernel weights does not match tensor dimensions)
[10/07/2021-16:49:44] [TRT] [E] 3: [CONVOLUTION #1] torch.nn.Conv2d.forward(Conv2d(3, 32, kernel_size=(7, 7), stride=(2, 2), padding=(3, 3)), tensor(shape=[2, 3, 184, 320], dtype=torch.float32)):kernel weights has count 4704 but 9408 was expected
[10/07/2021-16:49:44] [TRT] [E] 4: [CONVOLUTION #1] torch.nn.Conv2d.forward(Conv2d(3, 32, kernel_size=(7, 7), stride=(2, 2), padding=(3, 3)), tensor(shape=[2, 3, 184, 320], dtype=torch.float32)): count of 4704 weights in kernel, but kernel dimensions (7,7) with 6 input channels, 32 output channels and 1 groups were specified. Expected Weights count is 6 * 7*7 * 32 / 1 = 9408
[10/07/2021-16:49:44] [TRT] [E] 4: [convolutionNode.cpp::nvinfer1::builder::ConvolutionNode::computeOutputExtents::39] Error Code 4: Internal Error ([CONVOLUTION #1] torch.nn.Conv2d.forward(Conv2d(3, 32, kernel_size=(7, 7), stride=(2, 2), padding=(3, 3)), tensor(shape=[2, 3, 184, 320], dtype=torch.float32)): number of kernel weights does not match tensor dimensions)
[10/07/2021-16:49:44] [TRT] [E] 3: [CONVOLUTION #1] torch.nn.Conv2d.forward(Conv2d(3, 32, kernel_size=(7, 7), stride=(2, 2), padding=(3, 3)), tensor(shape=[2, 3, 184, 320], dtype=torch.float32)):kernel weights has count 4704 but 9408 was expected
[10/07/2021-16:49:44] [TRT] [E] 4: [CONVOLUTION #1] torch.nn.Conv2d.forward(Conv2d(3, 32, kernel_size=(7, 7), stride=(2, 2), padding=(3, 3)), tensor(shape=[2, 3, 184, 320], dtype=torch.float32)): count of 4704 weights in kernel, but kernel dimensions (7,7) with 6 input channels, 32 output channels and 1 groups were specified. Expected Weights count is 6 * 7*7 * 32 / 1 = 9408
[10/07/2021-16:49:44] [TRT] [E] 4: [convolutionNode.cpp::nvinfer1::builder::ConvolutionNode::computeOutputExtents::39] Error Code 4: Internal Error ([CONVOLUTION #1] torch.nn.Conv2d.forward(Conv2d(3, 32, kernel_size=(7, 7), stride=(2, 2), padding=(3, 3)), tensor(shape=[2, 3, 184, 320], dtype=torch.float32)): number of kernel weights does not match tensor dimensions)
[10/07/2021-16:49:44] [TRT] [E] 3: [CONVOLUTION #1] torch.nn.Conv2d.forward(Conv2d(3, 32, kernel_size=(7, 7), stride=(2, 2), padding=(3, 3)), tensor(shape=[2, 3, 184, 320], dtype=torch.float32)):kernel weights has count 4704 but 9408 was expected
[10/07/2021-16:49:44] [TRT] [E] 4: [CONVOLUTION #1] torch.nn.Conv2d.forward(Conv2d(3, 32, kernel_size=(7, 7), stride=(2, 2), padding=(3, 3)), tensor(shape=[2, 3, 184, 320], dtype=torch.float32)): count of 4704 weights in kernel, but kernel dimensions (7,7) with 6 input channels, 32 output channels and 1 groups were specified. Expected Weights count is 6 * 7*7 * 32 / 1 = 9408
[10/07/2021-16:49:44] [TRT] [E] 4: [convolutionNode.cpp::nvinfer1::builder::ConvolutionNode::computeOutputExtents::39] Error Code 4: Internal Error ([CONVOLUTION #1] torch.nn.Conv2d.forward(Conv2d(3, 32, kernel_size=(7, 7), stride=(2, 2), padding=(3, 3)), tensor(shape=[2, 3, 184, 320], dtype=torch.float32)): number of kernel weights does not match tensor dimensions)
[10/07/2021-16:49:44] [TRT] [E] 3: [CONVOLUTION #1] torch.nn.Conv2d.forward(Conv2d(3, 32, kernel_size=(7, 7), stride=(2, 2), padding=(3, 3)), tensor(shape=[2, 3, 184, 320], dtype=torch.float32)):kernel weights has count 4704 but 9408 was expected
[10/07/2021-16:49:44] [TRT] [E] 4: [CONVOLUTION #1] torch.nn.Conv2d.forward(Conv2d(3, 32, kernel_size=(7, 7), stride=(2, 2), padding=(3, 3)), tensor(shape=[2, 3, 184, 320], dtype=torch.float32)): count of 4704 weights in kernel, but kernel dimensions (7,7) with 6 input channels, 32 output channels and 1 groups were specified. Expected Weights count is 6 * 7*7 * 32 / 1 = 9408
[10/07/2021-16:49:44] [TRT] [E] 4: [convolutionNode.cpp::nvinfer1::builder::ConvolutionNode::computeOutputExtents::39] Error Code 4: Internal Error ([CONVOLUTION #1] torch.nn.Conv2d.forward(Conv2d(3, 32, kernel_size=(7, 7), stride=(2, 2), padding=(3, 3)), tensor(shape=[2, 3, 184, 320], dtype=torch.float32)): number of kernel weights does not match tensor dimensions)

Could you please help? Thank you
